Samples/aggregate-traffic-count-data-within-localauthorities.py

```diff
@@ -1,22 +1,38 @@
+'''
+  This recipe gets two datasources
+     - London Traffic data using TrafficCountImporter.java only for London
+     - London localAuthority data using OaImporter.java
+  
+  Then for each localAuthority using GeographicAggregationField
+    - It sums all the bi-cycle count based on LocalAuthority
+  
+  Then it writes the sum to the output file for every localAuthority using GeographicAggregationField
+'''
 from os import path, pardir
 import sys
 sys.path.append(path.join(path.dirname(path.realpath(__file__)), pardir))
 
 from recipe import Dataset, Subject, AttributeMatcher, GeographicAggregationField, LatestValueField, Match_Rule, Datasource, Recipe
```

Comment:
say sth like importing all the relevant objects that are needed

Reply:
done